mxcube3/ui/main.jsx

```diff
@@ -70,7 +70,7 @@ export default class App extends React.Component {
   componentWillMount() {
     const persistor = persistStore(store,
            { blacklist: ['remoteAccess', 'beamline', 'sampleChanger',
-                         'form', 'login', 'general', 'logger', 'points'],
+                         'form', 'login', 'general', 'logger', 'points', 'queue'],
```

Comment:
I know but I will leave this as it is until we have fixed the rest of PR. The reason being that this could hide issues with the init queue_ho.